index.html

```diff
@@ -216,25 +119,75 @@ <h3 class="project-title">
               alt=""
             />
           </div>
-          <div class="model-body">
-            <h2 class="model-text">Keeping track of hundreds of components</h2>
-            <div class="mobile-list">
-              <ul>
-                <li>Ruby on Rails</li>
-                <li>CSS</li>
-                <li>JavaScript</li>
-              </ul>
-            </div>
-            <div class="desktop-list">
-              <ul>
-                <li>Codekit</li>
-                <li>GitHub</li>
-                <li>JavaScript</li>
-                <li>Bootstrap</li>
-                <li>Terminal</li>
-                <li>Codepen</li>
-              </ul>
+          <div class="model-container">
+            <div class="model-body">
+              <h2 class="model-text">
+                Keeping track of hundreds of components
+              </h2>
+              <div class="mobile-list">
+                <ul>
+                  <li>Ruby on Rails</li>
+                  <li>CSS</li>
+                  <li>JavaScript</li>
+                </ul>
+              </div>
+
+              <div class="desktop-list">
+                <ul>
+                  <li>Codekit</li>
+                  <li>GitHub</li>
+                  <li>JavaScript</li>
+                  <li>Bootstrap</li>
+                  <li>Terminal</li>
+                  <li>Codepen</li>
+                </ul>
+              </div>
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry. Lorem Ipsum has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a galley
+                of type and scrambled it 1960s.
+              </p>
+              <p>
+                Lorem Ipsum is simply dummy text of the printing and typesetting
+                industry. Lorem Ipsum has been the industry's standard dummy
+                text ever since the 1500s, when an unknown printer took a galley
+                of type and scrambled it 1960s.
+              </p>
             </div>
+          </div>
+          <div class="model-footer desktop-flex">
+            <button class="btn-live">
+              See live
+              <span
+                ><i
+                  class="fa-thin fa-arrow-up-left-from-circle fa-rotate-90"
+                ></i
+              ></span>
+            </button>
+            <button class="btn-source">
+              See source <span><i class="fa-brands fa-github"></i></span>
+            </button>
+          </div>
+        </div>
+      </div>
+      <div id="overlay"></div>
+    </section>
+    <!-- modal section -->
+    <section id="model-section" class="model-section">
+      <div class="model">
+        <div class="model-content">
+          <div class="model-header">
+            <button class="model-close" type="button">X</button>
+            <img src="./images/Snapshoot Portfolio (1).png" alt="Model-image" />
```

Comment:
- [ ] As suggested by the previous reviewer, I would encourage you to remove the scrolling effect for a good user experience.

- [ ] The desktop and mobile versions should be exactly the same as the chosen design template in Figma.

> Figma
<img width="478" alt="Screenshot 2023-05-26 at 11 44 07" src="https://github.com/Umuhan/My-Portfolio-Project/assets/71140133/af348ee9-d458-4040-8c56-7dc7d06ac3e7">

> Your design
<img width="927" alt="Screenshot 2023-05-26 at 11 46 27" src="https://github.com/Umuhan/My-Portfolio-Project/assets/71140133/3eaa99b2-69d1-4d94-9262-d80ad4a7e892">